e2e/integration/helpers/helpers.go

```diff
@@ -0,0 +1,341 @@
+	}
+
+	return " ", resp.StatusCode
+}
+//Function to check the Pod state
+func CheckPodRunningState(apiserver string, podlist v1.PodList) {
+	Eventually(func() int {
+		var count int
+		for _, pod := range podlist.Items {
+			state, _:= GetPodState(apiserver+ "/"+pod.Name)
+			utils.InfoV2("PodName: %s PodStatus: %s", pod.Name, state)
+			if state == "Running"{
+				count ++
+			}
+		}
+		return count
+	}, "120s", "4s").Should(Equal(len(podlist.Items)), "Delete Application deployment is Unsuccessfull, Pod has not come to Running State")
+
+}
+
+//Function to check the Pod state
+func CheckPodDeleteState(apiserver string, podlist v1.PodList) {
+	Eventually(func() int {
+		var count int
+		for _, pod := range podlist.Items {
+			status, statusCode := GetPodState(apiserver+ "/"+pod.Name)
+			utils.InfoV2("PodName: %s status: %s StatusCode: %d", pod.Name, status, statusCode)
+			if statusCode == 404{
+				count ++
+			}
+		}
+		return count
+	}, "120s", "4s").Should(Equal(len(podlist.Items)), "Delete Application deployment is Unsuccessfull, Pod has not come to Running State")
+
+}
+
+func GenerateNodeReqBody(nodeid string) (error, map[string]interface{}){
+	var temp map[string]interface{}
+
+	body:= fmt.Sprintf(`{"kind": "Node","apiVersion": "v1","metadata": {"name": "%s","labels": {"name": "edgenode"}}}`, nodeid)
+	err:= json.Unmarshal([]byte(body), &temp)
+	if err!= nil{
+		utils.Failf("Unmarshal body failed: %v", err)
+		return err, temp
+	}
+
+	return nil, temp
+}
+
+//Function to get the pods from Edged
+func DeRegisterNodeFromMaster(ctx *utils.TestContext, nodehandler, nodename string) error {
+	err, resp := SenHttpRequest(http.MethodDelete, ctx.Cfg.ApiServer+nodehandler+ "/" +nodename)
+	if err != nil{
+		utils.Failf("Sending SenHttpRequest failed: %v", err)
+		return err
+	}
+	defer resp.Body.Close()
+	Expect(resp.StatusCode).Should(Equal(http.StatusOK))
+
+	return nil
+}
+
+//Function to get the pods from Edged
+func RegisterNodeToMaster(ctx *utils.TestContext, UID, nodehandler string) error {
+	err, body := GenerateNodeReqBody(UID)
+	if err != nil {
+		utils.Failf("Unmarshal body failed: %v", err)
+		return err
+	}
+
+	client := &http.Client{}
+	t := time.Now()
+	nodebody, err:= json.Marshal(body)
+	if err != nil{
+		utils.Failf("Marshal body failed: %v", err)
+		return err
+	}
+	BodyBuf := bytes.NewReader(nodebody)
+	req, err := http.NewRequest(http.MethodPost, ctx.Cfg.ApiServer+nodehandler, BodyBuf)
+	req.Header.Set("Content-Type", "application/json")
+	if err != nil {
+		utils.Failf("Frame HTTP request failed: %v", err)
+		return err
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		utils.Failf("Sending HTTP request failed: %v", err)
+		return err
+	}
+	utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+	defer resp.Body.Close()
+
+	Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+	return nil
+}
+
+func CheckNodeReadyStatus(ctx *utils.TestContext, nodehandler, nodename string) string{
```

Comment:
Pls add a comment with a short description above this function explaining its functionality

Reply:
Can we pls use camel case for nodehandler, nodename ??